CHANGELOG.md

```diff
@@ -19,6 +19,7 @@
 * IriConverter: Fix IRI url double encoding - may cause breaking change as some characters no longer encoded in output (#3552)
 * OpenAPI: **BC** Replace all characters other than `[a-zA-Z0-9\.\-_]` to `.` in definition names to be compliant with OpenAPI 3.0 (#3669)
 * Add stateless ApiResource attribute
+* Add TraversablePaginator
```

Comment:
Can you add it to a new `2.7.0` section please? We'll merge this new feature in the next version.